src/components/attendee/TableComponent/TableComponent.module.scss

```
@@ -70,7 +76,8 @@
   margin: space.empty(0.5);
   width: space.empty(4);
   height: space.empty(4);
-  border-radius: 100%;
+  border-radius: 50%;
```

The old border constants are still here:

```suggestion
  border-radius: border.$radius--mx;
```

---

thanks! done: https://github.com/sparkletown/sparkle/pull/2910/commits/afe06cdd59c668832f14435166c1d3c97fba4064